superset-frontend/src/explore/components/controls/VizTypeControl.jsx

```diff
@@ -119,7 +120,7 @@ const VizTypeControl = props => {
 
   useEffect(() => {
     if (showModal) {
-      searchRef?.current?.focus();
+      setTimeout(() => searchRef?.current?.focus(), 200);
```

Comment:
Curious about the need for the timeout, and if the 200 is based on anything (e.g. some transition timing) or if it's just a guess at some loading/drawing/mounting time.

Reply:
BUT... this fixes the auto-focusing of the input when the modal opens, and I LOVE it.

Reply:
https://github.com/vazco/uniforms/issues/279
The 200 is a common value found in other places of the application for rendering purposes.